### DISCUSSION TITLE
Friends | 友情链接

### DISCUSSION THREAD

**Discussion by yujinyan** [yujinyan/blog#178] [Comments]

**Comment by luyuhuang** [comment]
Hi, your blog looks good. May I exchange a link with you? My blog's address is <https://luyuhuang.tech>.

**Comment by yujinyan** [comment]
> Hi, your blog looks good. May I exchange a link with you? My blog's address is https://luyuhuang.tech.

@luyuhuang Sure, just added.

BTW, I noticed my name is misspelled on the friends page. 😉

Also, the profile pics appear blurry. A high-res version of mine can be found at the following link. Feel free to use that if you'd like things to look sharper.

https://github.com/yujinyan/blog/blob/master/content/assets/profile-pic.png

**Comment by luyuhuang** [comment]
Sorry, I have corrected it, and the avatar has been replaced too.

**Comment by Moosphan** [comment]
Your blog looks good, I'm also interested at Kotlin and android, maybe we can exchange a link.

**Comment by yujinyan** [comment]
> Your blog looks good, I'm also interested at Kotlin and android, maybe we can exchange a link.

@Moosphan Hi, just added your site (presuming it's https://dorck.cn/). I enjoy reading your blog ;-).

**Comment by Moosphan** [comment]
Thx :-)

**Comment by imjiaoyuan** [comment]
Hi,I think maybe my blog can link with you :https://yuanj.top
